src/common/contracts.py

```diff
@@ -291,13 +324,75 @@ async def aggregate(
         return await self.contract.functions.aggregate(data).call(block_identifier=block_number)
 
 
+class EigenPodOwnerContract(ContractWrapper):
+    abi_path = 'abi/IEigenPodOwner.json'
+
+    def __init__(self, address):
+        self._address = address
+
+    @property
+    def contract_address(self) -> ChecksumAddress:
+        return self._address
+
+    # pylint: disable-next=too-many-arguments
+    async def get_verify_and_process_withdrawals_call(
+        self,
+        oracle_timestamp: int,
+        state_root_proof: tuple[bytes, bytes],
+        validator_fields_proofs: list[bytes],
+        validator_fields: list[list[bytes]],
+        withdrawal_proofs: list[
+            tuple[
+                bytes,
+                bytes,
+                bytes,
+                bytes,
+                bytes,
+                int,
+                int,
+                int,
+                bytes,
+                bytes,
+                bytes,
+                bytes,
+            ]
+        ],
+        withdrawal_fields: list[bytes],
+    ) -> tuple[ChecksumAddress, HexStr]:
+        call = self.encode_abi(
+            fn_name='verifyAndProcessWithdrawals',
+            args=[
+                oracle_timestamp,
+                state_root_proof,
+                withdrawal_proofs,
+                validator_fields_proofs,
+                validator_fields,
+                withdrawal_fields,
+            ],
+        )
+
+        return self.address, call
```

Comment:
is returning the address actually necessary?
you can do `eigen_pod_owner_contract.address`

Reply:
We are commonly use `get_smth_call() -> tuple[ChecksumAddress, HexStr]:`  function signatures  in the operator service.  So that just to unify contract call prepartion